src/components/vm/disks/diskAdd.jsx

```diff
@@ -78,6 +78,12 @@ function getDiskUsageMessage(vms, storagePool, volumeName) {
     return message;
 }
 
+function getDefaultVolumeName(poolName, storagePools, vm) {
+    const vmStoragePool = storagePools.find(pool => pool.name == poolName);
+    const filteredVolumes = getFilteredVolumes(vmStoragePool, vm.disks);
+    return filteredVolumes[0] && filteredVolumes[0].name;
```

Comment:
This can return undefined. Is that expected and can callers deal with it?

Reply:
Yes, the return value of this is only used to set "existingVolumeName" state property, which can be `undefined` (it is even initialized as `undefined`)